src/css/components/files/_c-cart-body-scroll.scss

```diff
@@ -0,0 +1,5 @@
+.c-cart-body-scroll{
```

Comment:
The rules here look very generic and can be declared as utility rules so that they can be used elsewhere, if needed.